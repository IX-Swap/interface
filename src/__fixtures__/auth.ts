import { user } from './user'
import { APIServiceResponse } from 'v2/services/api/types'
import User from 'v2/types/user'
import {
  CompletePasswordResetArgs,
  LoginArgs,
  RequestPasswordResetArgs,
  SignupArgs,
  VerifySignupArgs
} from 'v2/types/auth'

const email = 'alex@investax.io'
const password = 'Qwerty12345_'

export const authURLs = {
  login: '/auth/sign-in',
  signup: '/auth/registrations',
  verifySignup: '/auth/registrations/confirm',
  getUser: '/auth/profiles',
  requestPasswordReset: '/auth/password/reset/start',
  completePasswordReset: '/auth/password/reset/confirm'
}

export const loginArgs: LoginArgs = {
  otp: '123456',
  password,
  email
}

export const signupArgs: SignupArgs = {
  name: 'Alex Solovev',
  email,
  password
}

export const verifySignupArgs: VerifySignupArgs = {
  verificationToken: 'mytoken'
}

export const requestPasswordResetArgs: RequestPasswordResetArgs = {
  email
}

export const completePasswordResetArgs: CompletePasswordResetArgs = {
  email,
  resetToken: 'token',
  newPassword: password
}

export const loginResponseSuccess: APIServiceResponse<User> = {
  data: user,
  message: 'Welcome',
  success: true
}

export const loginResponseFailure: APIServiceResponse<User> = {
  message: 'error',
  success: false
}

export const signupResponseSuccess: APIServiceResponse = {
  message: 'A verification e-mail has been sent to your account',
  success: true
}

export const signupResponseFailure: APIServiceResponse = {
  message: 'Sorry but this email address is already taken',
  success: false
}

export const verifySignupResponseSuccess: APIServiceResponse = {
  message: 'Your account has been verified',
  success: true
}

export const verifySignupResponseFailure: APIServiceResponse = {
  message: 'The token has already been verified, has expired, or is invalid',
  success: false
}

export const getUserResponseSuccess: APIServiceResponse<User> = {
  data: user,
  message: 'OK',
  success: true
}

export const getUserResponseFailure: APIServiceResponse<User> = {
  message: 'Access denied',
  success: false
}

export const requestPasswordResetResponseSuccess: APIServiceResponse = {
  message: 'OK',
  success: true
}

export const requestPasswordResetResponseFailure: APIServiceResponse = {
  message: 'Error',
  success: false
}

export const completePasswordResetResponseSuccess: APIServiceResponse = {
  message: 'Success',
  success: true
}

export const completePasswordResetResponseFailure: APIServiceResponse = {
  message: 'Reset token is already used, expired, or invalid',
  success: false
}
