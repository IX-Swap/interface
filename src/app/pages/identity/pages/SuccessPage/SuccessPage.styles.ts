import makeStyles from '@mui/styles/makeStyles'

export const useStyles = makeStyles(theme => ({
  box: {
    width: theme.spacing(115),
    height: theme.spacing(46.5),
    backgroundColor: theme.palette.background.paper,
    borderRadius: theme.spacing(2.1),
    display: 'flex',
    flexDirection: 'row'
  },
  boxAlert: {
    width: theme.spacing(115),
    backgroundColor: theme.palette.background.paper,
    borderRadius: theme.spacing(2.1),
    display: 'flex',
    flexDirection: 'row',
    [theme.breakpoints.down('md')]: {
      width: theme.spacing(43),
      flexDirection: 'column',
      borderRadius: 0,
      padding: theme.spacing(2.5)
    }
  },
  pageHeader: {
    [theme.breakpoints.down('md')]: {
      display: 'none'
    }
  },
  iconSuccess: {
    width: theme.spacing(18.6),
    margin: theme.spacing(8, 0, 0, 7.2),
    [theme.breakpoints.down('md')]: {
      width: '303px',
      display: 'flex',
      justifyContent: 'center',
      margin: 0
    }
  },
  topText: {
    [theme.breakpoints.down('md')]: {
      fontSize: theme.spacing(3)
    }
  },
  icon2fa: {
    width: theme.spacing(17.8),
    margin: theme.spacing(8, 0, 0, 7.9),
    [theme.breakpoints.down('md')]: {
      width: '303px',
      display: 'flex',
      justifyContent: 'center',
      margin: 0
    }
  },
  mainContent: {
    marginTop: theme.spacing(8),
    height: theme.spacing(29),
    width: theme.spacing(60),
    [theme.breakpoints.down('md')]: {
      width: theme.spacing(38.1),
      textAlign: 'center',
      marginTop: theme.spacing(1.5)
    }
  },
  button: {
    width: theme.spacing(15.6),
    marginTop: theme.spacing(3),
    padding: '16px !important',
    boxShadow: 'none',
    [theme.breakpoints.down('md')]: {
      width: theme.spacing(38)
    }
  },
  text: {
    fontSize: theme.spacing(2),
    fontWeight: 400,
    lineHeight: theme.spacing(3),
    opacity: '80%',
    [theme.breakpoints.down('md')]: {
      lineHeight: theme.spacing(2.5),
      fontSize: theme.spacing(1.7)
    }
  }
}))
