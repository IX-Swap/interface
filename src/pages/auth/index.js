import React, { Suspense } from 'react';
import { Switch, Route, Redirect } from 'react-router-dom';

import SignIn from './pages/sign-in';
import Reset from './pages/reset';

const Confirm = React.lazy(() => import('./pages/confirm'));

export default function Auth() {
  return (
    <Suspense fallback={<div>Loading...</div>}>
      <Switch>
        <Route path="/auth/sign-in" exact component={SignIn} />
        <Route path="/auth/reset" exact component={Reset} />
        <Route path="/auth/confirm" exact component={Confirm} />

        <Route path="/auth" render={() => <Redirect to="/auth/sign-in" />} />
      </Switch>
    </Suspense>
  );
}
