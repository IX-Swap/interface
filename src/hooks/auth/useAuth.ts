import { useCachedUser } from 'hooks/auth/useCachedUser'

export const useAuth = () => {
  const user = useCachedUser()

  return {
    isAuthenticated: true,
    user: user
  }
}

export default useAuth
