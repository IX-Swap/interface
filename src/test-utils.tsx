import React from 'react'
import { Router } from 'react-router-dom'
import { render, RenderOptions, RenderResult } from '@testing-library/react'
import {
  createGenerateClassName,
  StylesProvider
} from '@material-ui/core/styles'
import { ThemeProvider } from '@material-ui/styles'
import Themes from './v2/themes'
import { history } from 'v2/history'
import { UserProvider } from 'v2/auth/context'
import { UserStore } from 'v2/auth/context/store'
import { PasswordResetProvider } from 'v2/auth/context/password-reset'
import PasswordResetStore from 'v2/auth/context/password-reset/store'
import { PasswordResetStep } from 'v2/auth/context/password-reset/types'
import { AuthorizerTableStore } from 'v2/app/pages/authorizer/context/store'
import { AuthorizerTableStoreProvider } from 'v2/app/pages/authorizer/context'
import { DepositStore } from 'v2/app/pages/accounts/pages/banks/context/store'
import { DepositStoreProvider } from 'v2/app/pages/accounts/pages/banks/context'
import { ServicesProvider } from 'v2/services/useServices'
import { renderHook, RenderHookResult } from '@testing-library/react-hooks'
import { BreadcrumbsProvider } from 'v2/hooks/useBreadcrumbs'

const generateClassName = createGenerateClassName({
  productionPrefix: 'ix'
})

const BaseProviders: React.FC = ({ children }) => {
  return (
    <StylesProvider generateClassName={generateClassName}>
      <ThemeProvider theme={Themes.default}>
        <BreadcrumbsProvider>
          <Router history={history}>{children}</Router>
        </BreadcrumbsProvider>
      </ThemeProvider>
    </StylesProvider>
  )
}

const customRenderer = (
  ui: any,
  options?: Omit<RenderOptions, 'queries'>
): RenderResult => {
  return render(ui, { wrapper: BaseProviders, ...options })
}

export const testComponentId = 'test-component'

export const fakeUserStore: Partial<UserStore> = {
  activeTab: 0,
  setActiveTab: jest.fn()
}

export const renderWithUserStore = (
  ui: any,
  store?: Partial<UserStore>
): RenderResult => {
  const WithUserProvider: React.FC = ({ children }) => (
    <BaseProviders>
      <UserProvider value={{ ...fakeUserStore, ...store }}>
        {children}
      </UserProvider>
    </BaseProviders>
  )

  return render(ui, { wrapper: WithUserProvider })
}

export const fakePasswordResetStore: Partial<PasswordResetStore> = {
  currentStep: PasswordResetStep.Request,
  setCurrentStep: jest.fn()
}

export const renderWithPasswordResetStore = (
  ui: any,
  store?: Partial<PasswordResetStore>
): RenderResult => {
  const WithUserProvider: React.FC = ({ children }) => (
    <BaseProviders>
      <PasswordResetProvider value={{ ...fakePasswordResetStore, ...store }}>
        {children}
      </PasswordResetProvider>
    </BaseProviders>
  )

  return render(ui, { wrapper: WithUserProvider })
}

export const fakeAuthorizerTableStore: Partial<AuthorizerTableStore> = {
  idKey: '__',
  uri: '/api/source',
  _getItemId: jest.fn()
}

export const renderWithAuthorizerTableStore = (
  ui: any,
  store?: Partial<AuthorizerTableStore>
): RenderResult => {
  const WithUserProvider: React.FC = ({ children }) => (
    <BaseProviders>
      <AuthorizerTableStoreProvider
        value={{ ...fakeAuthorizerTableStore, ...store }}
      >
        {children}
      </AuthorizerTableStoreProvider>
    </BaseProviders>
  )

  return render(ui, { wrapper: WithUserProvider })
}

export const fakeDepositStore: Partial<DepositStore> = {}

export const renderWithDepositStore = (
  ui: any,
  store?: Partial<DepositStore>
): RenderResult => {
  const WithUserProvider: React.FC = ({ children }) => (
    <BaseProviders>
      <DepositStoreProvider value={{ ...fakeDepositStore, ...store }}>
        {children}
      </DepositStoreProvider>
    </BaseProviders>
  )

  return render(ui, { wrapper: WithUserProvider })
}

export const renderHookWithServiceProvider = (
  hookFn: any,
  store: object = {}
): RenderHookResult<any, any> => {
  const WithServiceProvider: React.FC = ({ children }) => (
    <ServicesProvider value={store}>{children}</ServicesProvider>
  )

  return renderHook(hookFn, { wrapper: WithServiceProvider })
}

export * from '@testing-library/react'
export { customRenderer as render }
