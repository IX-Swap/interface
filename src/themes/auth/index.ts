import { createTheme } from '@mui/material/styles'
import { darkTheme } from 'themes/auth/dark'
import { getThemeOverrides } from 'themes/auth/overrides'
import { typography } from 'themes/auth/typography'

export enum AppTheme {
  Dark = 'Dark'
}

export const getAppTheme = () => {
  const theme = createTheme({ ...darkTheme, typography })

  theme.components = getThemeOverrides(theme)

  return theme
}
