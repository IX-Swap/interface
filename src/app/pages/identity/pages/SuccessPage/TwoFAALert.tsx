import React from 'react'
import { useStyles } from 'app/pages/identity/pages/SuccessPage/SuccessPage.styles'
import { Button, Grid, Typography } from '@mui/material'
import { Box } from '@mui/system'
import { useAuth } from 'hooks/auth/useAuth'
import { ReactComponent as Enabled } from 'assets/icons/2fa/security-enabled.svg'
import { ReactComponent as Disabled } from 'assets/icons/2fa/security-disabled.svg'
import { AppRouterLinkComponent } from 'components/AppRouterLink'
import { useAppBreakpoints } from 'hooks/useAppBreakpoints'
import { SecurityRoute } from 'app/pages/security/router/config'

export const TwoFAALert = () => {
  const classes = useStyles()
  const { isMobile } = useAppBreakpoints()
  const { user = { enable2Fa: undefined } } = useAuth()
  const { enable2Fa } = user
  const isProtected = enable2Fa !== undefined && enable2Fa
  return (
    <Box>
      {isProtected ? (
        <Box
          className={classes.boxAlert}
          height={isMobile ? 300 : 284}
          mb={isMobile ? 2 : 10}
        >
          <Grid className={classes.icon2fa}>
            <Enabled
              height={isMobile ? 73 : 100}
              width={isMobile ? 60 : 81.82}
            />
          </Grid>
          <Grid className={classes.mainContent}>
            <Typography variant='h2' className={classes.topText}>
              Account Protected
            </Typography>
            <Typography className={classes.text} mt={2}>
              Your account is now protected by Two-Factor Authentication
            </Typography>
            <Button
              className={classes.button}
              variant='outlined'
              size='large'
              to={SecurityRoute.change2fa}
              component={AppRouterLinkComponent}
            >
              Update 2FA
            </Button>
          </Grid>
        </Box>
      ) : (
        <Box className={classes.boxAlert} height={326} mb={isMobile ? 2 : 10}>
          <Grid className={classes.icon2fa}>
            <Disabled
              height={isMobile ? 73 : 100}
              width={isMobile ? 60 : 81.82}
            />
          </Grid>
          <Grid className={classes.mainContent}>
            <Typography variant='h2' className={classes.topText}>
              Protect your account!
            </Typography>
            <Typography className={classes.text} mt={2} color='#F56283'>
              Your account is currently at risk as Two Factor Authentication
              (2FA) is not enabled now. We urge you to activate this feature as
              soon as posible.
            </Typography>
            <Button
              className={classes.button}
              to={SecurityRoute.change2fa}
              component={AppRouterLinkComponent}
              variant='contained'
              size='large'
            >
              Enable 2FA
            </Button>
          </Grid>
        </Box>
      )}
    </Box>
  )
}
