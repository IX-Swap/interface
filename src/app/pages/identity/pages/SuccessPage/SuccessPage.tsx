import React from 'react'
import { useStyles } from 'app/pages/identity/pages/SuccessPage/SuccessPage.styles'
import { Button, Grid, Typography } from '@mui/material'
import { Box } from '@mui/system'
import { ReactComponent as Success } from 'assets/icons/alerts/success.svg'
import { IdentityRoute } from 'app/pages/identity/router/config'
import { TwoFAALert } from 'app/pages/identity/pages/SuccessPage/TwoFAALert'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { RootContainer } from 'ui/RootContainer'
import { useAuth } from 'hooks/auth/useAuth'
import { AppRouterLinkComponent } from 'components/AppRouterLink'
import { useAppBreakpoints } from 'hooks/useAppBreakpoints'

export const SuccessPage = () => {
  const classes = useStyles()
  const data = useAuth()
  const { isMobile } = useAppBreakpoints()
  const name = data.user?.name

  return (
    <Grid container>
      <Grid item xs={12} className={classes.pageHeader}>
        <PageHeader title={name} />
      </Grid>
      <RootContainer style={{ paddingTop: 0 }}>
        <Box
          className={classes.boxAlert}
          mb={2}
          height={372}
          mt={isMobile ? 5 : 0}
        >
          <Grid className={classes.iconSuccess}>
            <Success height={isMobile ? 60 : 80} width={isMobile ? 60 : 80} />
          </Grid>
          <Grid className={classes.mainContent}>
            <Typography variant='h2' className={classes.topText}>
              Thank you for your application!
            </Typography>
            <Typography className={classes.text} mt={2}>
              We are checking your application now, this usually takes from one
              to three days
            </Typography>
            <Typography className={classes.text} mt={2}>
              If you have any further questions, you can always contact our{' '}
              <a href=' https://investax.io/contact/ '>support deprtment</a>
            </Typography>
            <Button
              component={AppRouterLinkComponent}
              to={IdentityRoute.list}
              variant='outlined'
              size='large'
              replace={false}
              disableElevation
              className={classes.button}
            >
              View Identity
            </Button>
          </Grid>
        </Box>
        <TwoFAALert />
      </RootContainer>
    </Grid>
  )
}
