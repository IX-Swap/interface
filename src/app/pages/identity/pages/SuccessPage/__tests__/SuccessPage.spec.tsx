import React from 'react'
import { SuccessPage } from 'app/pages/identity/pages/SuccessPage/SuccessPage'
import { render } from 'test-utils'
import * as useAuth from 'hooks/auth/useAuth'
import { TwoFAAlert } from 'app/pages/identity/pages/SuccessPage/TwoFAAlert'

jest.mock('app/pages/identity/pages/SuccessPage/TwoFAAlert', () => ({
  TwoFAAlert: jest.fn(() => null)
}))
describe('Checking TwoFAAlert component', () => {
  it('checking 2fa Alert props', () => {
    const userData = {
      isAuthenticated: true,
      user: {
        enable2Fa: undefined
      }
    }

    jest.spyOn(useAuth, 'useAuth').mockImplementation(() => userData as any)

    render(<TwoFAAlert />)
    expect(TwoFAAlert).toHaveBeenCalled()
  })
})

describe('Snapshot test for Success Page', () => {
  it('should check SuccessPage after render', () => {
    const page = render(<SuccessPage />)
    expect(page).toMatchSnapshot()
  })
})
