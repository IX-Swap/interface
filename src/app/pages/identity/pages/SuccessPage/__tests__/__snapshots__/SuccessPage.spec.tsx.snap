// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Success Page checking TwoFA alert with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div />
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot test for Success Page should check SuccessPage after render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 makeStyles-pageHeader-3 css-ewf807-MuiGrid-root"
        >
          <div
            class="makeStyles-wrapper-15 MuiBox-root css-0"
          >
            <div
              class="MuiContainer-root MuiContainer-maxWidthLg css-127nfur-MuiContainer-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column makeStyles-container-13 css-1vpwcmr-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container makeStyles-header-12 css-11lq3yg-MuiGrid-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item css-i6prtc-MuiGrid-root"
                  >
                    <h2
                      class="MuiTypography-root MuiTypography-h2 makeStyles-title-16 css-rldqdm-MuiTypography-root"
                    />
                  </div>
                </div>
                <div
                  class="MuiGrid-root css-vj1n65-MuiGrid-root"
                >
                  <nav
                    class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-128kl2r-MuiTypography-root-MuiBreadcrumbs-root"
                  >
                    <ol
                      class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
                    />
                  </nav>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-p03apy-MuiContainer-root"
          style="padding-top: 0px;"
        >
          <div
            class="makeStyles-boxAlert-2 MuiBox-root css-1pxauac"
          >
            <div
              class="MuiGrid-root makeStyles-iconSuccess-4 css-vj1n65-MuiGrid-root"
            />
            <div
              class="MuiGrid-root makeStyles-mainContent-7 css-vj1n65-MuiGrid-root"
            >
              <h2
                class="MuiTypography-root MuiTypography-h2 makeStyles-topText-5 css-rldqdm-MuiTypography-root"
              >
                Thank you for your application!
              </h2>
              <p
                class="MuiTypography-root MuiTypography-body1 makeStyles-text-9 css-e3fhh0-MuiTypography-root"
              >
                We are checking your application now, this usually takes from one to three days
              </p>
              <p
                class="MuiTypography-root MuiTypography-body1 makeStyles-text-9 css-e3fhh0-MuiTypography-root"
              >
                If you have any further questions, you can always contact our
                 
                <a
                  href=" https://investax.io/contact/ "
                >
                  support department
                </a>
              </p>
              <a
                class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-disableElevation MuiButtonBase-root makeStyles-button-8 css-1dz9je6-MuiTypography-root-MuiLink-root-MuiButtonBase-root-MuiButton-root"
                href="/app/identity"
                tabindex="0"
              >
                View Identity
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 makeStyles-pageHeader-3 css-ewf807-MuiGrid-root"
      >
        <div
          class="makeStyles-wrapper-15 MuiBox-root css-0"
        >
          <div
            class="MuiContainer-root MuiContainer-maxWidthLg css-127nfur-MuiContainer-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column makeStyles-container-13 css-1vpwcmr-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container makeStyles-header-12 css-11lq3yg-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item css-i6prtc-MuiGrid-root"
                >
                  <h2
                    class="MuiTypography-root MuiTypography-h2 makeStyles-title-16 css-rldqdm-MuiTypography-root"
                  />
                </div>
              </div>
              <div
                class="MuiGrid-root css-vj1n65-MuiGrid-root"
              >
                <nav
                  class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-128kl2r-MuiTypography-root-MuiBreadcrumbs-root"
                >
                  <ol
                    class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
                  />
                </nav>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-p03apy-MuiContainer-root"
        style="padding-top: 0px;"
      >
        <div
          class="makeStyles-boxAlert-2 MuiBox-root css-1pxauac"
        >
          <div
            class="MuiGrid-root makeStyles-iconSuccess-4 css-vj1n65-MuiGrid-root"
          />
          <div
            class="MuiGrid-root makeStyles-mainContent-7 css-vj1n65-MuiGrid-root"
          >
            <h2
              class="MuiTypography-root MuiTypography-h2 makeStyles-topText-5 css-rldqdm-MuiTypography-root"
            >
              Thank you for your application!
            </h2>
            <p
              class="MuiTypography-root MuiTypography-body1 makeStyles-text-9 css-e3fhh0-MuiTypography-root"
            >
              We are checking your application now, this usually takes from one to three days
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 makeStyles-text-9 css-e3fhh0-MuiTypography-root"
            >
              If you have any further questions, you can always contact our
               
              <a
                href=" https://investax.io/contact/ "
              >
                support department
              </a>
            </p>
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeLarge MuiButton-outlinedSizeLarge MuiButton-disableElevation MuiButtonBase-root makeStyles-button-8 css-1dz9je6-MuiTypography-root-MuiLink-root-MuiButtonBase-root-MuiButton-root"
              href="/app/identity"
              tabindex="0"
            >
              View Identity
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
